sisyphus.pl

```diff
@@ -432,7 +432,6 @@ =head1 DESCRIPTION
 check_errs \$? "bcl2fastq failed in $fastqPath/IndexCheck"
 
 checkIndices.pl -runfolder '$rfPath' -demuxSummary '$rfPath/IndexCheck/Stats' 
-check_errs \$? "IndexCheck failed"
```

Comment:
it would be good to save the exit status and do the bailout after generating the quick report instead of just skipping the check